ansible/playbook.yml

```diff
@@ -1,9 +1,18 @@
 - hosts: all
 
   pre_tasks:
+    - name: Check if running in wsl 2
+      ansible.builtin.shell: wsl.exe -- ls
+      ignore_errors: true
+      changed_when: true
+      register: wsl_is_wsl_command
+    - name: Set facts if running in wsl
+      ansible.builtin.set_fact:
+        wsl_is_wsl: '{{ true if (wsl_is_wsl_command.rc == 0) else false }}'
```

Comment:
Move this whole block into a separate tasks file similar to the below ones.

Reply:
done